utils/flakiness-dashboard/processing/dashboard_compressed_v1.js

```diff
@@ -60,7 +60,7 @@ function compressReports(reports) {
         specs = new Map();
         files[spec.file] = specs;
       }
-      const specId = spec.file + '---' + spec.title + ' --- ' + spec.line;
+      const specId = spec.file + '---' + spec.titlePath.join('-') + ' --- ' + spec.line;
```

Comment:
Can we use same path delimiter when joining (it was `>` in https://github.com/microsoft/playwright/pull/24486) ?

Reply:
Make sure titlePath does not include project